### ISSUE TITLE
Add custom attributes

### ISSUE THREAD

**Issue by Spyna** [HashLips/hashlips_art_engine#1192]
I'd like to add a feature to generate custom random attributes.

**Comment by bolshoytoster** [comment]
@Spyna can you not just have a layer of fully transparent images with the attribute names?

**Comment by Spyna** [comment]
If you want to range between 100 values (maybe with numeric values) you don't want to create 100 files.

**Comment by Spyna** [comment]
I created the pull request https://github.com/HashLips/hashlips_art_engine/pull/1193 to add this feature

**Comment by bolshoytoster** [comment]
@Spyna fair enough, but your PR is unlikely to be merged because HashLips has been inactive for a few months.

Also, you shouldn’t create an extra file (`src/attributes.config.js`), just put the extra config in `src/config.js`.

And in [`addCustomMetadata`](https://github.com/Spyna/hashlips_art_engine/blob/05ccc4662b33efaa53d9925fb3bda41362b5a463/src/main.js#L180-L196) you could add some extra spacing, at least a space after the `if` and before the `else`.

**Comment by Spyna** [comment]
How come the project is at a standstill? Is there another one active? 

It is better to add another small file that does one thing instead of adding more functionality to an already large file. It's called the single responsibility principle in programming. 

For the spacing, I followed the style of the rest of the code, in which there are no spaces after ifs

**Comment by bolshoytoster** [comment]
@Spyna 
> How come the project is at a standstill?

HashLips is busy with another project

> Is there another one active?

There’s [my fork](https://github.com/bolshoytoster/hashlips_art_engine), I try to stay as compatible with the main repo as possible, and there are a few other forks as well.

> For the spacing, I followed the style of the rest of the code, in which there are no spaces after ifs

Out of 32 `if`s in the repo, all of them have spaces after them.

**Comment by Spyna** [comment]
Oh sorry, you're right. Spaces, not newlines :sweat_smile:. I added the commit just now for the spaces. 
If your project is active and in use, feel free to create a pull request from my fork or copy the sources directly to it.

**Comment by bolshoytoster** [comment]
@Spyna
> Oh sorry, you're right. Spaces, not newlines 😅

Sorry, that was probably my fault.

I’ve opened [a PR](https://github.com/bolshoytoster/hashlips_art_engine/pull/6) for my fork, I’ll resolve the conflicts when I get home.
